scripts/pagination-test-wallets.js

```diff
@@ -0,0 +1,152 @@
+const { protocol: { consensus } } = require('bcoin');
+
+module.exports = async (node, config, logger, wallet) => {
+  consensus.COINBASE_MATURITY = 0;
+
+  const miner = node.miner;
+  const chain = node.chain;
+  const network = node.network;
+  const feeRate = network.minRelay * 10; // for some reason bc segwit??!!
+  const wdb = wallet.wdb;
+
+  const numInitBlocks = 144 * 3; //. to activate segwit
```

Comment:
No PR yet, just a head-scratching issue, I could spend more time figuring it out but I don't know if that's a priority since we can just bump the fee:
https://github.com/bcoin-org/bcoin/issues/666

Reply:
I was talking about the activation rather than the fee: https://github.com/bcoin-org/bcoin/pull/580

Reply:
Oh I see! Well there ya go, still a PR. It's ok though just means there's more blocks mined at startup, more funds to disperse... You could bump `numTxBlocks` and `numTxPerBlock` if you want to spread that dough around and mke more txs to paginate...